config/default.yml

```diff
@@ -995,6 +995,10 @@ Layout/ParameterAlignment:
 
 Layout/RescueEnsureAlignment:
   Description: 'Align rescues and ensures correctly.'
+  EnforcedStyleAlignWith: keyword
```

Comment:
Why not simply `EnforcedStyle`?

Reply:
It looked like the alignment cops were using `EnforcedStyleAlignWith`, so I was just trying to be consistent with them.  Why not simply `EnforcedStyle` for `Layout/BlockAlignment`, `Layout/DefEndAlignment`, `Layout/EndAlignment` and `Layout/RescueEnsureAlignment`?  I'm fine either way.  Did you want me to change it to `EnforcedStyle`?

Reply:
> It looked like the alignment cops were using `EnforcedStyleAlignWith`

That is my experience/expectation as well, FWIW

Reply:
I no longer remember why we did it like this in the past, but I definitely don't like it today. Probably it was to avoid styles named `align_with_*`, but I think that's fine.